hack/install-golangcilint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+GOLANGCILINT_VERSION=${GOLANGCILINT_VERSION:-v1.21.0}
+
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)/bin" "${GOLANGCILINT_VERSION}"
```

Comment:
lets NOT use master... lets pick a version or commit sha

Reply:
That's what `GOLANGCILINT_VERSION` does! We install tag `v1.21.0` using [install.sh](https://github.com/golangci/golangci-lint/blob/master/install.sh).

Reply:
great... lets lock the bootstrapping script to a sha

Reply:
Oh, I see, it's about the source of the `install.sh` script, not the version of the binary.